### ISSUE TITLE
DXT1A Wrong threating of Alpha Channel (CompressonatorCLI)

### ISSUE THREAD

**Issue by rbt2008** [GPUOpen-Tools/Compressonator#12]
Hi,

I have a texture in tga format with alpha channel and I want to compress it to DXT1 with 1bit alpha.

Following command:
```bat
CompressonatorCLI.exe -fd DXT1 -DXT1UseAlpha 1 -nomipmap source.tga compressed.dds
```
Will produce DXT1 image without alpha at all.

Following command:
```bat
CompressonatorCLI.exe -fd DXT1 -DXT1UseAlpha 1 -AlphaThreshold 255 -nomipmap source.tga compressed.dds
```
however, will give me desired result.

As far as I know the black color in alpha mask is "0", so it's surely should be less then "128", but somehow it's not working and such values are ignored.

Source texture that was used: http://s000.tinyupload.com/download.php?file_id=02204180364423075394&t=0220418036442307539465752

Maybe I'm missing something?

Thanks!

**Comment by NPCompress** [comment]
Thanks for the source file and comment on the issue,
We are investigating it now and will report back ASAP

**Comment by NPCompress** [comment]
It seems that the AlphaThreshold default value 128 gets reset to 0 when the code is run multithreaded (by default). 
You will have to specify a  -AlphaThreshold   value when enabling -DXT1UseAlpha

**Comment by rbt2008** [comment]
Ok, thank you!

Is it planned to fix or -AlphaThreshold should be always specified?

Thanks!

**Comment by NPCompress** [comment]
It will be fixed in future version updates, I would recommend you continue to use the -AlphaThreshold even if it has been fixed to use default.

**Comment by xooiamd** [comment]
fixed on commit 59bc8f306340b745922a9e46ea665e4e67da54aa
